G/GMP/build_tarballs.jl

```diff
@@ -4,27 +4,20 @@ using BinaryBuilder
 
 # Collection of sources required to build GMP
 name = "GMP"
-version = v"6.1.2"
+version = v"6.2.0"
 
 sources = [
     ArchiveSource("https://gmplib.org/download/gmp/gmp-$(version).tar.bz2",
-                  "5275bb04f4863a13516b2f39392ac5e272f5e1bb8057b18aec1c9b79d73d8fb2"),
+                  "f51c99cb114deb21a60075ffb494c1a210eb9d7cb729ed042ddb7de9534451ea"),
     DirectorySource("./bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd $WORKSPACE/srcdir/gmp-*
 
-# Update config.status
-update_configure_scripts
-
-# Patch `configure` to include `$LDFLAGS` in its tests.  This is necessary on FreeBSD.
-atomic_patch -p1 ${WORKSPACE}/srcdir/patches/configure.patch
```

Comment:
I removed this as in a local test, everything seemed to still build just fine on FreeBSD with GMP 6.2.0. Perhaps that means whatever was the actual issue that this patch worked around got resolve (in GMP or in BinaryBuilder or...). But since I don't know what the problem was, I can't be sure... So perhaps there is something subtly broken by removing this? For reference, this patch was added by @staticfloat in 58ed8ebd32be076a17efa5a4cd04bb0dc0143834.

Reply:
Seems this comes from https://github.com/JuliaLang/julia/pull/31586 by @vtjnash ; his report to GMP upstream can be read [here](https://gmplib.org/list-archives/gmp-bugs/2019-March/004520.html) with a follow up [here](https://gmplib.org/list-archives/gmp-bugs/2019-March/004523.html) which says:
> The failure we observe is:
> 
> > could not find a working compiler, see config.log for details

But since the FreeBSD builds are passing for this PR, I feel some confidence now that removing this is fine.

Reply:
We used to need to include LDFLAGS everywhere for FreeBSD, but now with the compiler wrappers the relevant flags are always passed

Reply:
Ahh, that explains it. Thank you!